### PR TITLE
task: Implement function to return registered attendees for an event

### LINKED ISSUE #21: Function that return number event registration
Given an event Id, return the total number of people that register for an event,

Write unit test for the function

### PR DESCRIPTION
**Pull request by Abeeujah**
closes #21 

This changes implements the following
- Function to return registered attendees for a given event (event_id)
- Tests to assert correctness

### CODE REVIEW

File: src/events/events.cairo [hunk 1]
```diff
@@ -169,5 +170,8 @@ pub mod Events {
             };
             event_attendance_details
         }
+        fn registered_attendees(self: @ContractState, event_id: u256) -> u256 {
```
Comment (mubarak23) [inline src/events/events.cairo:173]: can you ensure the function caller is the event organizer

File: tests/test_contract.cairo [hunk 2]
```diff
@@ -45,3 +45,11 @@ fn test_add_event() {
     stop_cheat_caller_address(event_contract_address);
 }
 
+#[test]
+fn test_registered_attendees() {
```
Comment (mubarak23) [inline tests/test_contract.cairo:49]: prank that event owner is caller of this function

File: tests/test_contract.cairo [hunk 3]
```diff
@@ -45,3 +45,11 @@ fn test_add_event() {
     stop_cheat_caller_address(event_contract_address);
 }
```
Comment (mubarak23) [inline tests/test_contract.cairo:47]: add a should panic test for case where the wrong owner call the function

File: Scarb.lock [hunk 4]
```diff
@@ -95,7 +95,15 @@ name = "openzeppelin_utils"
 version = "0.15.0"
 source = "git+https://github.com/OpenZeppelin/cairo-contracts.git?tag=v0.15.0#f57642960f1c8cffafefb88bfff418eca8510634"
```
Comment (mubarak23) [inline Scarb.lock:97]: remove this scarb.lock file

**Review by mubarak23 — CHANGES_REQUESTED** [review]
also pull from the main branch to resolve the merge conflict on the PR.

**Comment by mubarak23** [comment]
@Abeeujah any progress regarding this review requested

File: src/events/events.cairo (hunk 1 above)
Comment (Abeeujah) [reply src/events/events.cairo]: Done!

File: tests/test_contract.cairo (hunk 2 above)
Comment (Abeeujah) [reply tests/test_contract.cairo]: Done!

File: tests/test_contract.cairo (hunk 3 above)
Comment (Abeeujah) [reply tests/test_contract.cairo]: Done!

File: Scarb.lock (hunk 4 above)
Comment (Abeeujah) [reply Scarb.lock]: Done!

**Comment by Abeeujah** [comment]
All requested changes has been included @mubarak23 :rocket:

**Pull request merged**